src/libs/common/ide/types/Events.ts

```diff
@@ -0,0 +1,46 @@
+import type { TextDocument, Range } from "@cursorless/common";
```

Comment:
Given the amount of interface definitions / docstrings we've stolen from VSCode in this directory, we might want to put a note in the directory to that effect, and pointing to the VSCode license

Reply:
Do just mean put a read me there or a license file or?